projects.yaml

```diff
@@ -555,3 +555,7 @@ projects:
         - *module_if
         - *module_hic_stm32f103xb
         - records/board/ublox_evk_odin_w2.yaml
+    lpc11u35_pitaya_link_if:
```

Comment:
Can you move this to keep the interface projects sorted?

Reply:
Resolved as per comment